hoover/contrib/twofactor/models.py

```diff
@@ -3,8 +3,8 @@
 from django.conf import settings
 from django.db import models
 
-VOCABULARY = u'abcdefghijklmnopqrstuvwxyz0123456789'
-REQUIRED_ENTROPY = 40
+VOCABULARY = 'abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789'
```

Comment:
I would've used `string.ascii_lowercase + string.ascii_uppercase` and the like, but maybe it's clearer to type the whole alphabet every time...